README.md

```
@@ -1,36 +1,425 @@
+    <p>
+      &bull; 찜한 모임, 모든 리뷰, 마이페이지-내가 만든 모임 페이지 기능 구현<br/>
+      &bull; Local storage를 이용하여 모임의 찜하기 및 삭제 기능<br/>
+      &bull; React-query 의 InfiniteQuery 를 사용한 무한스크롤 기능 구현<br/>
+      &bull; 다크모드 디자인 적용<br/>
+    </p>
+  </div>
+  
+  </div>
+</details>
+
+##### 개발 기간: 2025.09 - 2025.11
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h5

(MD001, heading-increment)